Directory.Build.props

```diff
@@ -20,7 +20,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0; net7.0; net462; net472; netstandard2.0; netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetNet8)' == 'True'">net6.0; net7.0; net8.0; net462; net472; netstandard2.0; netcoreapp3.1</TargetFrameworks>
```

Comment:
>netcoreapp3.1 [](http://example.com/codeflow?start=116&length=13)

@jmprieur when do we want to remove this?

Reply:
TBD